rust_team_data/src/v2.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+pub static BASE_URL: &str = "https://team-api.infra.rust-lang.org/v2";
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct Permission {
+    pub people: Vec<PermissionPerson>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord)]
+pub struct PermissionPerson {
+    pub github_id: usize,
+    pub github: String,
+    pub name: String,
+    pub discord_id: Option<usize>,
```

Comment:
Can we remove the discord ID until some tool actually needs it?

Reply:
are you sure that no tool is using it? why was it added to the v1 API if nothing is using it? 😅